app/wasm_config.go

```diff
@@ -63,7 +77,59 @@ func NewDesmosCustomQueryPlugin(
 	querier := wasmdesmos.NewQuerier(queriers)
 
 	return wasm.QueryPlugins{
-		Custom: querier.QueryCustom,
+		Stargate: wasmkeeper.AcceptListStargateQuerier(GetStargateAcceptedQueries(), grpcQueryRouter, cdc),
```

Comment:
Allowing all desmos custom moduels' querier for stargate feature. Our queries do not have non-deterministic issue.

Note: cosmos-sdk stargate querier has non-deterministic issue since some of the queries allow to retrieve historical data, like `staking.HistoricalInfo`. The result historical queries will be influenced by validator pruning setting, as a result, occurs non-deterministic issue.